tests/acceptance/features/apiCollaboration/checkFileInfo.feature

```diff
@@ -599,3 +599,620 @@ Feature: check file info with different wopi apps
       | Collabora     |
       | FakeOffice    |
       | OnlyOffice    |
+
+
+  Scenario Outline: try to get file info of deleted file with office suites
```

Comment:
Please, add the issue tag and let's wait for the fix. I see that the fix PR is ready